dhall-try/index.html

```diff
@@ -242,30 +263,26 @@ <h1 class="display-4">Turing-completeness is not a feature</h1>
       to do so as a specification bug.</p>
       <a href="https://github.com/dhall-lang/dhall-lang/wiki/Safety-guarantees"class="btn btn-lg btn-outline-dark bg-light">Safety Guarantees<i class="fas fa-shield-alt"></i></a>
     </div>
-    <div class="container jumbotron" style="background-color: #cfebfb">
-      <h1 class="display-4">Integration friendly</h1>
-      <p class="lead">Dhall provides a smooth migration path for mature enterprise deployments.</p>
-      <hr class="my-4">
-      <p>You can convert both ways between Dhall and JSON/YAML or read Dhall
-      configuration files directly into a language that supports a native
-      language binding.</p>
-      <a href="https://github.com/dhall-lang/dhall-lang/wiki/How-to-integrate-Dhall" class="btn btn-lg btn-outline-dark bg-light">Integrations <i class="fas fa-cogs" ></i></a>
-    </div>
     <div class="container jumbotron" style="background-color: #fbf3c8">
-      <h1 class="display-4">Tired of YAML?</h1>
-      <p class="lead">Dhall eliminates the YAML quirks that software engineers
-      dread.  The language standard learns from the mistakes of the past.</p>
+      <h1 class="display-4">Use a real programming language</h1>
+      <p class="lead">Tired of being a "YAML engineer"?  Migrate your existing
+      YAML (or JSON) files to Dhall
+      <hr class="my-4">
       <textarea id="highlight1"></textarea>
-      <p>You can also generate either YAML or JSON from Dhall configuration
-      files using tiny statically-linked binaries.  Dhall can cheaply fit right
-      in with your existing tools and infrastructure.</p>
+      <blockquote class="blockquote">
+      <p>"Configuration bugs, not code bugs, are the most common cause I've seen of really bad outages. When I looked at publicly available postmortems, searching for “global outage postmortem” returned about 50% outages caused by configuration changes. Publicly available postmortems aren't a representative sample of all outages, but a random sampling of postmortem databases also reveals that config changes are responsible for a disproportionate fraction of extremely bad outages. As with error handling, I'm often told that it's obvious that config changes are scary, but it's not so obvious that most companies test and stage config changes like they do code changes."</p>
```

Comment:
@sjakobi: How about keeping the first and last sentence, like this:

> "Configuration bugs, not code bugs, are the most common cause I've seen of really bad outages. … As with error handling, I'm often told that it's obvious that config changes are scary, but it's not so obvious that most companies test and stage config changes like they do code changes."

The last sentence is really the key one that I want to presreve because the theme of this section is to treat configuration file changes as seriously as we treat code changes.

Reply:
Unfortunately I find that last sentence kind of hard to understand. So I'm not sure whether it makes for a good quote. But I'm a non-native speaker so maybe the target demographic will get more out of it than I do.

Reply:
I'm pretty confident that the target audience of this section (people who work on ops / shared infrastructure / CI) will understand what it means.